orderbook/src/database.rs

```diff
@@ -57,4 +57,45 @@ impl Database {
             .await?;
         Ok(())
     }
+
+    async fn count_rows_in_table(&self, table: &str) -> Result<i64> {
+        let query = format!("SELECT COUNT(*) FROM {};", table);
```

Comment:
Apparently you can't use `$1` in the from position.

Reply:
I am glad this isn't a public method because I suppose if `table = "orders; DROP TABLE orders;"` we could get in trouble here. 

In general, I am under the impression that we should try and avoid making database queries with variable string inputs. Do we have an enum somewhere with table names or could we generate one with a DB query:

```sh
postgres=# \dt
```

Reply:
You should definitely never create sql queries from strings with attacker controlled inputs. In this case we control the possible values for `table` so it's fine.